packs/default/structures/single/village_house_desert.yml

```diff
@@ -0,0 +1,27 @@
+files:
+  village_house_desert: 1
+id: "VILLAGE_HOUSE_DESERT"
+spawn:
+  start:
+    min: 72
+    max: 72
+  bound:
+    min: 48
+    max: 96
+  width: 175
+  padding: 50
+loot:
+  0: village_house_desert
+features:
+  - type: ENTITY_FEATURE
+    entity: VILLAGER
+    attempts: 200
+    in-height: 2
+    amount:
+      min: 4
+      max: 4
+    spawnable-on:
```

Comment:
```yml
    spawnable-on:
      - "minecraft:cobblestone"
      - "minecraft:grass_block"
      - "minecraft:oak_stairs"
      - "minecraft:oak_slab"
    spawnable-in:
      - "minecraft:air"
      - "minecraft:green_carpet"
      - "minecraft:grass"
      - "minecraft:tall_grass"
```

^ Drastically increases chances of successful spawn.

Reply:
This village house is for the Desert, which consists of green carpet, sand, sandstone, and such?

Reply:
Ah, my bad - should be adapted for that, then. This is just an example of what I did to increase spawn chance in the oak one.

Something like this:
```yml
    spawnable-on:
      - "minecraft:sandstone"
      - "minecraft:sand"
      - "minecraft:sandstone_slab"
    spawnable-in:
      - "minecraft:air"
      - "minecraft:green_carpet"
      - "minecraft:dead_bush"
      - "minecraft:acacia_button"
      - "minecraft:stone_button"
```